experimental/apps-mcp/cmd/init_template.go

```diff
@@ -0,0 +1,98 @@
+package mcp
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/experimental/apps-mcp/lib/middlewares"
+	"github.com/databricks/cli/experimental/apps-mcp/lib/session"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/template"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultTemplateRepo = "https://github.com/neondatabase/appdotbuild-agent"
+	defaultTemplateDir  = "edda/edda_templates/trpc_bundle"
+)
+
+func newInitTemplateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "init-template PROJECT_NAME",
+		Short: "Initialize a new app from template",
+		Long: `Initialize a new Databricks app from a template.
+
+This is a shortcut for 'bundle init' with the default MCP app template.
+Auto-detects the SQL warehouse ID unless DATABRICKS_WAREHOUSE_ID is set.`,
```

Comment:
Let's also make `--warehouse-id` an optional parameter. I can imagine that eventually we want the agent to keep track of which warehouse to use and pass it explicitly.